### ISSUE TITLE
not working recently

### ISSUE THREAD

**Issue by kodiaddontest** [YoColdRain/Hotstar-Rain#18]
thank you for great add on, but recently its not working. can you please check.

**Comment by ibshar** [comment]
Yes, it suddenly stopped working few days back. Can you please check.

**Comment by shubhsheth** [comment]
finally got it to work 
https://github.com/shubhsheth/Hotstar-Rain
get the v1.0.15

**Comment by ibshar** [comment]
Actually on further investigation found that the issue was with an unofficial version of the plugin(**1.0.16**). So when I installed the official **v1.0.14a** it worked fine.

**Comment by shubhsheth** [comment]
Actually I was having issue too so I tried fixing it myself and now it works for me. So just for others who have same issue

**Comment by nandunair75** [comment]
v1.0.14 working perfectly

**Comment by gunavignesh** [comment]
this addon is not working to me, even i tried 1.0.14, as well tried 1.0.15 from https://github.com/shubhsheth/Hotstar-Rain. But nothing gets working, also i tried by entering indian IP in options. So kindly update, if any solution exist already to this issue.
